contracts/voting-verifier/src/execute.rs

```diff
@@ -161,7 +164,78 @@ pub fn verify_messages(
         .into(),
     ))
 }
+struct PollResults(Vec<Option<Vote>>);
+
+// would be better to implement the Sub trait, but clippy is configured to not allow arithmetic operators
+impl PollResults {
+    /// Returns the elements in self that are Some, but in rhs are None. All other elements are converted to None.
+    /// This is used to determine which elements have quorum in self, but do not have quorum in rhs.
+    /// Vectors must be equal length.
+    fn sub(self, rhs: Self) -> Result<PollResults, ContractError> {
```

Comment:
```suggestion
    fn difference(self, rhs: Self) -> Result<PollResults, ContractError> {
```
see https://doc.rust-lang.org/std/collections/struct.HashSet.html#method.difference for example